GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -53,7 +53,9 @@ protected override void Execute(GVFSEnlistment enlistment)
                 this.WriteMessage(GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath());
                 this.WriteMessage(string.Empty);
                 this.WriteMessage("Enlistment root: " + enlistment.EnlistmentRoot);
-                this.WriteMessage("Cache Server: " + CacheServerResolver.GetUrlFromConfig(enlistment));
+                string cacheServer = CacheServerResolver.GetUrlFromConfig(enlistment);

```

Comment:
Note that `GetUrlFromConfig` should have returned the origin URL if there was no cache server configured. So this is not just a case of "no cache server". Did we actually somehow create a bogus origin remote? Or is there some other bug that is causing that bad URL to leak out of this method?